exports.js

```diff
@@ -806,6 +806,7 @@ module.exports = {
         'logRetentionDays'              : require(__dirname + '/plugins/azure/postgresqlserver/logRetentionDays.js'),
         'connectionThrottlingEnabled'   : require(__dirname + '/plugins/azure/postgresqlserver/connectionThrottlingEnabled.js'),
         'logDurationEnabled'            : require(__dirname + '/plugins/azure/postgresqlserver/logDurationEnabled.js'),
+        'postgresqlCMKEncrypted'        : require(__dirname + '/plugins/azure/postgresqlserver/postgresqlEncryptionAtRestWithCMK.js'),
```

Comment:
```suggestion
        'postgresqlCMKEncrypted'        : require(__dirname + '/plugins/azure/postgresqlserver/postgresqlEncryptionAtRestWithCMK.js'),
```
change filename to 'postgresqlCMKEncrypted'